commit/merkleroot/rmn/controller.go

```diff
@@ -247,29 +247,29 @@ func (c *controller) Close() error {
 	return c.peerClient.Close()
 }
 
-// getRmnSignedObservations guarantees to return at least #minObservers signed observations for each source chain.
+// getRmnSignedObservations guarantees to return at least f+1 signed observations for each source chain.
 func (c *controller) getRmnSignedObservations(
 	ctx context.Context,
 	destChain *rmnpb.LaneDest,
 	updateRequestsPerChain map[uint64]updateRequestWithMeta,
 	configDigest cciptypes.Bytes32,
-	minObserversMap map[cciptypes.ChainSelector]int,
+	homeFMap map[cciptypes.ChainSelector]int,
 	rmnNodeInfo map[rmntypes.NodeID]rmntypes.HomeNodeInfo,
 ) ([]rmnSignedObservationWithMeta, error) {
 	requestedNodes := make(map[uint64]mapset.Set[rmntypes.NodeID])                   // sourceChain -> requested rmnNodeIDs
 	requestsPerNode := make(map[rmntypes.NodeID][]*rmnpb.FixedDestLaneUpdateRequest) // grouped requests for each node
 
-	// For each lane update request send an observation request to at most 'minObservers' number of rmn nodes.
-	// At this point we can safely assume that we have at least #minObservers supporting each source chain.
+	// For each lane update request send an observation request to at most 'f+1' number of rmn nodes.
+	// At this point we can safely assume that we have at least f+1 supporting each source chain.
 	for sourceChain, updateRequest := range updateRequestsPerChain {
 		requestedNodes[sourceChain] = mapset.NewSet[rmntypes.NodeID]()
-		minObservers, exist := minObserversMap[cciptypes.ChainSelector(sourceChain)]
+		homeF, exist := homeFMap[cciptypes.ChainSelector(sourceChain)]
```

Comment:
```suggestion
		chainF, exist := homeFMap[cciptypes.ChainSelector(sourceChain)]
```

Or similar, this reads as if it is the F value of the home chain, not the appropriate source chain.